src/components/EntityInfo.vue

```diff
@@ -69,6 +69,7 @@
 
               <!-- Dissolve Company -->
               <span v-if="isAllowed(AllowableActions.DISSOLVE_COMPANY)">
+              <!-- <span v-if="isAllowedFiling(FilingTypes.DISSOLUTION)"> -->
```

Comment:
The changes in here are for the subject ticket -- not cleanup but actual requirements :)

This is WIP but doesn't affect anything.